test/box/net.box_reuse_tuple_formats_gh-6217.result

```diff
@@ -11,7 +11,7 @@ box.schema.user.grant('guest', 'execute', 'universe')
  | ...
 
 -- Check that formats created by net.box for schema are reused (gh-6217).
-COUNT = 113
+COUNT = 200
```

Comment:
Better submit unrelated changes like this in separate PRs - they will be merged faster.

I'd also write a more descriptive commit subject because "test 6217" looks confusing.